dist/leaflet.css

```diff
@@ -431,7 +431,7 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-top: none;
 	line-height: 1.1;
 	padding: 2px 5px 1px;
-	font-size: 11px;
+	font-size: 0.69rem;

```

Comment:
@Chandu-4444 would you like to change this to also include the `px` unit declarations as a fallback per https://github.com/Leaflet/Leaflet/pull/7571#issuecomment-961001947?

Reply:
Yeah sure!